torch/csrc/jit/codegen/cuda/graph_fuser.cpp

```diff
@@ -2042,6 +2042,9 @@ void replaceAliasOpsWithCopy(std::shared_ptr<Graph>& graph, Block* block) {
         graph->insertNode(graph->create(op_mapping[n->kind()], n->inputs(), 1));
     op_copy->output()->setType(n->output(0)->type());
 
+    // adding newly created value into alias_db;
+    alias_db->createValue(op_copy->output());
```

Comment:
Do we need to remove the value from `alias_db` in `revertAliasCopyOps`?
https://github.com/csarofeen/pytorch/blob/85eceb82e0b0a39c8d92f869ae1b6be42198833d/torch/csrc/jit/codegen/cuda/graph_fuser.cpp#L2056

Reply:
I don't think that's necessary. We'll just leave a dangling value in the set and that shouldn't hurt anything?